ProjectSettings/QualitySettings.asset

```diff
@@ -4,7 +4,7 @@
 QualitySettings:
   m_ObjectHideFlags: 0
   serializedVersion: 5
-  m_CurrentQuality: 5
+  m_CurrentQuality: 0
```

Comment:
What's this change?

Reply:
Cause I was testing on HoloLens it might have changed the quality to the lowest.